var/spack/repos/builtin/packages/hip/package.py

```diff
@@ -48,6 +48,24 @@ class Hip(CMakePackage):
     # See https://github.com/ROCm-Developer-Tools/HIP/pull/2141
     patch('0002-Fix-detection-of-HIP_CLANG_ROOT.patch', when='@3.5.0:')
 
+    def setup_run_environment(self, env):
+        env.set('ROCM_PATH', '')
+        env.set('HIP_COMPILER', 'clang')
+        env.set('HIP_PLATFORM', 'hcc')
+        env.set('HIP_CLANG_PATH', self.spec['llvm-amdgpu'].prefix.bin)
+        env.set('HSA_PATH', self.spec['hsa-rocr-dev'].prefix)
+        env.set('ROCMINFO_PATH', self.spec['rocminfo'].prefix)
+        env.set('DEVICE_LIB_PATH', self.spec['rocm-device-libs'].prefix.lib)
```

Comment:
```suggestion
        env.set('DEVICE_LIB_PATH', self.spec['rocm-device-libs'].libs.directories[0])
```
In case libs are installed into lib64 instead of lib